templates/ci_model/com.redhat.viaq-ci.template.json

```diff
@@ -98,6 +93,11 @@
               "index": "not_analyzed",
               "type": "string"
             },
+            "number": {
+              "doc_values": true,
+              "index": "not_analyzed",
```

Comment:
Do we need this for a string type?

Reply:
@portante Did you mean to ask "Do we need this for an *integer* type?"
<https://www.elastic.co/guide/en/elasticsearch/reference/2.4/number.html>